dev-lang/64tass/64tass-1.55.recipe

```diff
@@ -0,0 +1,68 @@
+SUMMARY="Multi pass optimizing macro assembler for the 65xx series of processors"
+DESCRIPTION="64tass is cross assembler targeting the 65xx series of micro processors \
+which features the familiar syntax of Omicron TASS and TASM."
+HOMEPAGE="https://sourceforge.net/projects/tass64/"
+COPYRIGHT="2013-2020 Soci/Singular
+	1997-2001 Benjamin Sittler"
+LICENSE="GNU LGPL v2.1
+	my_getopt"
+REVISION="1"
+SOURCE_URI="https://github.com/irmen/64tass/archive/v$portVersion.2202.tar.gz"
+CHECKSUM_SHA256="c9222531a711cf7f23fe0294dd953c2a7cda37f84070fdaeaf8c83a7284cc582"
+SOURCE_DIR="64tass-$portVersion.2202"
+
+ARCHITECTURES="!x86_gcc2 x86_64 ?arm ?ppc ?sparc"
+SECONDARY_ARCHITECTURES="x86"
+
+commandSuffix=$secondaryArchSuffix
+commandBinDir=$binDir
+if [ "$targetArchitecture" = x86_gcc2 ]; then
+	commandSuffix=
+	commandBinDir=$prefix/bin
+fi
+
+portVersionCompat="$portVersion compat >= ${portVersion%%.*}"
+
+PROVIDES="
+	64tass$secondaryArchSuffix = $portVersionCompat
+	cmd:64tass$secondaryArchSuffix = $portVersionCompat"
+REQUIRES="
+	haiku$secondaryArchSuffix
+	"
+
+BUILD_REQUIRES="
+	haiku${secondaryArchSuffix}_devel
+	"
+BUILD_PREREQUIRES="
+	cmd:gcc$secondaryArchSuffix
+	cmd:gzip
+	cmd:make
+	cmd:svnversion
```

Comment:
svnversion shouldn't be needed.

Reply:
If i understood correctly, I think it is instead. It is used to check for revision number inside the makefile. It has no effect actually (let alone I get the source from a git mirror) but will break the build if absent.

Reply:
The makefile has to be fixed then.